### DISCUSSION TITLE
kairship/instance/index

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2307] [Announcements]
# kairship/instance/index

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/instance/index.html

**Comment by liyuerich** [comment]
实例信息展示截图需要更新，目前还提供了 管理面集群

**Comment by ModetaNiu** [comment]
Fixed in PR #5124 
The current page : https://docs.daocloud.io/kairship/instance/
